Sources/PackageLoading/ModuleMapGeneration.swift

```diff
@@ -106,7 +106,6 @@ extension ClangModule {
         case .directory(let path):
             output += "    umbrella \"\(path)\"\n"
         }
-        output += "    link \"\(c99name)\"\n"
```

Comment:
I think it is probably better to try to stick with using auto-linking; I'm not 100% confident about that, but let us try to stick with it for now.